### ISSUE TITLE
Error 508

### ISSUE THREAD

**Issue by hostingnuggets** [cmallwitz/Financials-Extension#110]
Hi, I just installed this extension in Calc (LibreOffice 7.4.7.2) on Debian 12 but unfortunately all examples below from the README:

```
    =GETREALTIME("IBM",21,"YAHOO")
    =GETREALTIME("IBM:NYQ",21,"FT")
    =GETREALTIME("EURUSD","LAST_PRICE","FT")
    =GETHISTORIC("IBM",90,"2020-12-01","YAHOO")
    =GETREALTIME("ETH-USD","LAST_PRICE","COINBASE")
```

Show me an `Error 508`. Any ideas what could be wrong?

**Comment by cmallwitz** [comment]
Odd - Error 508 - assuming this is a web server error 508 - would indicate
you have overloaded the target web server somehow. Not very likely across
all of Yahoo/FT/Coinbase.

Are you behind a corporate firewall that somehow could interfere with the
extension?

**Comment by hostingnuggets** [comment]
> Are you behind a corporate firewall that somehow could interfere with the extension?

No, home connection in Europe. Could be maybe some country restriction...

**Comment by hostingnuggets** [comment]
@cmallwitz I actually found the issue: instead of using `,` as a separator like in the example I needed to use a `;`.

But now I have another issue, for the following

```
=GETREALTIME("EUR=X";"LAST_PRICE";"YAHOO")
```

I get the following error:

```
Yahoo.getRealtime(EUR=X, 21) - crumb missing
```

Any ideas what's wrong here?
